triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceTest.scala

```diff
@@ -559,4 +561,93 @@ trait AbstractTriggerServiceTestAuthMiddleware
         _ <- resp.status shouldBe StatusCodes.Forbidden
       } yield succeed
   }
+
+  it should "request a fresh token after expiry on user request" in withTriggerService(Nil) {
+    uri: Uri =>
+      for {
+        resp <- listTriggers(uri, alice)
+        _ <- resp.status shouldBe StatusCodes.OK
+        // Expire old token and test the trigger service transparently requests a new token.
```

Comment:
I assume by ‘trigger service transparently requests a new token” you mean that the user gets directed through an auth flow on the middleware? We’re not actually refreshing a token here right?

Reply:
Yes, this is not using the refresh endpoint.